bundles/org.openhab.binding.openuv/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -129,6 +130,13 @@
 			</parameter>
 		</config-description>
 	</channel-type>
+
+	<channel-type id="elevation">
+		<item-type>Number:Angle</item-type>
+		<label>Elevation</label>
+		<description>The elevation of the sun</description>
+		<state readOnly="true" pattern="%.2f %unit%" />
```

Comment:
Readonly? The readme states that this is an input.

Reply:
Well spotted.